### PR TITLE
make way to open img lib with an img already open

### PR DESCRIPTION
**Pull request by chrischalstrom**
## Merge Checklist
- [ ] Code reviewed by a frontend lead (Steve Kinney, Alex Wicks, Chris Chalstrom, Alex Thomsen)

This doesn't change the UI/UX at all, it's just adding a way to open the image library with the details pane already open.

### CODE REVIEW

File: test_image/ImageStoryshots.ts [hunk 1]
```diff
@@ -22,10 +22,15 @@ const getMatchOptions = ({ context: { kind, story }, url }) => {
   // };
 };
 
+// our components sometimes setState on mount, thus we ought to allow a short delay
+// before the screenshot is captured for consistency's sake
+const beforeScreenshot = () => new Promise(resolve => setTimeout(resolve, 300));
```
Comment (chrischalstrom) [inline test_image/ImageStoryshots.ts]: without this the snapshot outcome was random.

File: src/image-library-details-pane.tsx [hunk 2]
```diff
@@ -45,15 +44,21 @@ export class ImageLibraryDetailsPane extends Component<
...
+            <h3>Upload: </h3>
+            <p className="small">{dateFormatter(image.created)}</p>
+          </div>
+        )}
 
-        <div className={Styles['metadata-wrap']}>
-          <h3>Dimensions: </h3>
-          <p className="small">{`${image.width} x ${image.height}`} px</p>
-        </div>
+        {image.width &&
```
Comment (stevekinney) [inline src/image-library-details-pane.tsx]: I might go with an abstraction here. `hasWidthAndHeight()`? `hasSize()`?

File: src/image-library.tsx [hunk 3]
```diff
@@ -98,7 +110,7 @@ export class ImageLibrary extends Component<
     );
   }
 
-  private onThumbnailClick = (selectedImage: SGLibraryImage) => {
+  private onImageSelected = (selectedImage: SGLibraryImage) => {
```
Comment (stevekinney) [inline src/image-library.tsx]: This is a better name.

File: src/image-library-details-pane.tsx (hunk 2 above)
Comment (chrischalstrom) [reply src/image-library-details-pane.tsx]: +1

**Pull request merged**